docker/main/ngen/Dockerfile

```diff
@@ -906,8 +908,16 @@ COPY --chown=${USER} --from=build_bmi_sac_sma /dmod/ /dmod/
 COPY --chown=${USER} --from=build_bmi_snow_17 /dmod/ /dmod/
 
 USER root
-# Update path and make sure dataset directory is there
-RUN echo "export PATH=${PATH}" >> /etc/profile \
+# TODO: (later) consider something like this in the future (at least optionally) when downloading the mc client below:
+# ARG MINIO_CLIENT_RELEASE="RELEASE.2024-07-08T20-59-24Z"
+# ...
+# https://dl.min.io/client/mc/release/linux-amd64/archive/mc.${MINIO_CLIENT_RELEASE}
+
+# Setup minio client; also update path and make sure dataset directory is there
+RUN curl -L -o /dmod/bin/mc https://dl.min.io/client/mc/release/linux-amd64/mc \
```

Comment:
```suggestion
RUN ARCHITECTURE=`echo $(uname -s)-$(uname -m) | tr '[:upper:]' '[:lower:]'`; \
curl -L -o /dmod/bin/mc https://dl.min.io/client/mc/release/${ARCHITECTURE}/mc \
```

Reply:
Hmm ... good catch, but the suggestion won't work as needed with Linux on an Intel machine (it produces `linux-x86_64` instead of `linux-amd64`).  I will make an adjustment, but I'll need to think more about exactly how.

Reply:
That wasn't the only case that would have been a problem, but I've made a change now that I think should catch the platform types we can reasonably expect that would need to be transformed to something else for purposes of that URL, and done the adjustment accordingly.